docs/installation.md

```diff
@@ -0,0 +1,293 @@
+# Installation
+
+This document provides detailed instructions on how to install the project. Follow these steps to set up the project on your local system.
+
+## Prerequisites
+
+[Homebrew](https://brew.sh) is the simplest way to install system dependencies needed for this project.
+
+[Chocolatey](https://community.chocolatey.org/) is a good equivalent to Homebrew for package management in Windows.
+
+## System Requirements
+
+
+- **Node/NPM** (recommend using nvm)
+- **MariaDB** (using MariaDB in docker is also good)
+- **Java** (for Liquibase / OpenAPI  Generator)
+- **Liquibase**
+- **OpenAPI Generator**
+- **pyenv**
+- **wget** *(optional)*
+
+### Mac
+
+```bash
+brew install nvm
+brew install java
+brew install liquibase
+brew install pyenv
+brew install wget
+
+# skip if you wish to install via docker
+brew install mariadb@10.8
+
+```
+
+### Windows
+
+```bash
+# Assuming you have Chocolatey
+choco install nvm
+choco install jdk8
+choco install liquibase
+choco install pyenv-win
+choco install wget
+
+# skip if you wish to install via docker
+choco install mariadb --version=10.8.3
+```
+
+```bash
+# Install npm
+nvm install --lts
+
+# Once you have npm set up, install the OpenAPI generator:
+npm install @openapitools/openapi-generator-cli -g
+openapi-generator-cli version-manager set 5.3.0
+```
+
+## Installation Steps
+
+### Creating the environment
+
+- Python dependencies for the `metamist` API package are listed in `setup.py`.
+- Additional dev requirements are listed in `requirements-dev.txt`.
+- Packages for the sever-side code are listed in `requirements.txt`.
+
+We *STRONGLY* encourage the use of `pyenv` for managing Python versions. Debugging and the server will run on a minimum python version of 3.10.
```

Comment:
Could we link back to the team-docs python page here?

Reply:
Done!